.github/workflows/submit.yml

```diff
@@ -7,21 +7,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Cache pnpm modules
+      - name: Set up Node.js
+        uses: actions/setup-node@v3.4.1
+        with:
+          node-version: "20.x"
+          cache: "yarn"
+      - name: Cache yarn modules
         uses: actions/cache@v3
         with:
-          path: ~/.pnpm-store
-          key: ${{ runner.os }}-${{ hashFiles('**/pnpm-lock.yaml') }}
+          path: "**/node_modules"
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
-            ${{ runner.os }}-
-      - uses: pnpm/action-setup@v2.2.1
-        with:
-          version: 7.1.0
-          run_install: true
-      - name: Build and zip extension artifact
-        run: pnpm build -- --zip
+            ${{ runner.os }}-yarn-
+      - name: Install dependencies
+        run: yarn install
+      - name: Create .env file
+        run: echo "PLASMO_PUBLIC_ONRAMPER_API_KEY=${{ secrets.ONRAMPER_API_KEY }}" >> .env
+      - name: Build and Package the extension into a zip artifact
+        run: yarn build:chrome --zip
       - name: Browser Platform Publish
-        uses: PlasmoHQ/bpp@v2
+        uses: PlasmoHQ/bpp@v3
         with:
-          keys: ${{ secrets.SUBMIT_KEYS }}
+          keys: ${{ secrets.SUBMIT_KEYS_STAGING }}
```

Comment:
Later on we should change this depending on the branch we are submiting, or maybe create another workflow for production